app/styles/layouts/_l-default.scss

```diff
@@ -159,10 +159,12 @@ body {
   @include breakpoint(large) {
     position: absolute;
     top: 10px;
-    left: 50px;
+    // left: 50px;
+    right: 20px;
     z-index: 4;
     padding-bottom: 0;
-    margin-top: rem-calc(92);
+    // margin-top: rem-calc(92);
```

Comment:
I'd just remove this sort of thing than keep it commented out

Reply:
Clearly I didn't do a great job of scrubbing out all of my unnecessary comments...